gallery/src/pages/StylingComponents.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {GalleryTile} from '../gallery-components/GalleryTiles';
+
+const StylingRSCPage = () =>
+  <GalleryTile title="How to style your app when using RSC">
+    <p className="nx-p">
+      If you have styling needs that aren't covered by the styles included in React Shared Components (RSC),
+      then you're going to have to create your own styles.
+    </p>
+    <h2 className="h2">Page specific stylesheets</h2>
+    <p className="nx-p">
+      Take a look at <code className="nx-code">frontend/src/main/scss/app-styles.scss</code>. This file
```

Comment:
Is this a file in the Sonatype Application Builder?  It doesn't appear to be a file in RSC.

Reply:
> Is this a file in the Sonatype Application Builder? It doesn't appear to be a file in RSC.

Switched reference to `main.scss`: https://github.com/sonatype/sonatype-react-shared-components/pull/72/commits/b3c842e717876bc3dfea76fdddb05203eadf2e55